README.md

```diff
@@ -138,25 +238,29 @@ This will be a delicate principle to consider, as we also will need to hold firm
 For example the ae project will need to read from a configuration file in order to communicate with a remote server. In one case we could assume that the configuration file is always in a familiar location for convenience. In the other case we do not want to force a user to pass a very long flag every time they run the tool. What do we do?
 
 We hold the opinion that the the file should be in a few well documented locations, and create logic to try the locations if no flag is passed. If a user passes a flag we should remove all assumptions from the program and only consider the input. We prefer clarity over magic.
-
-### Remote Servers instead of Local Systems
-
+    
+### **Remote Servers instead of Local Systems!**
+    
 This is an interesting principle to consider. As the functionality of the ae tool grows we will inevitably need to execute logic somewhere.
 
 We prefer to keep the ae tool as "boring" and "unintelligent" as possible. We do not want the command line tool to do a lot of "work" or "processing" locally. In situations where we need to perform logic against a set of Aurae nodes, we need to find a way to take the logic out of ae for compatibility reasons.
 
 We do not want to get into a situation where we have multiple clients attempting to perform slightly different tasks locally against a server. Especially when these clients are subject to take long periods of time for their work to complete.
 
-In the event these types of situations arise, it is a sign that we likely need to deploy a scheduler or server mechanisms somewhere to manage the work on our behalf.
-
-
-
-## Code Quality
-
-### Linting
-
-We are using the [golangci-lint](https://golangci-lint.run/) tool to lint the code. You will need to install it for your system, and can find instructions at [this link](https://golangci-lint.run/usage/install/). This tool runs on every pull request, and it must pass before merging is allowed. You can run it locally with `make lint`
-
-### Formatting
+In the event these types of situations arise, it is a sign that we likely need to deploy a scheduler or server mechanisms somewhere to manage the work on our behalf.    
+    
+<!-- CONTRIBUTE -->
+<h2 id="about-the-project">Contribute</h2>
```

Comment:
another id to update

Reply:
it occurs to me that you may not need these ids. github markdown creates anchors by default for headings: https://stackoverflow.com/questions/27981247/github-markdown-same-page-link

Reply:
solved in 8d446b6.
should work completly normal with less HTML. :)